features/append/append.feature

```diff
@@ -1,11 +1,11 @@
 Feature: append a new feature branch to an existing feature branch
 
   Background:
-    Given my repo has a feature branch "existing"
+    Given a feature branch "existing"
```

Comment:
The old version of this step was misleading. It doesn't create a branch in the user's local repo but also in origin.